### ISSUE TITLE
List Indices must be Integers

### ISSUE THREAD

**Issue by control-led** [JohannBraun/WoocommerceToGoogleSheets#1]
`  while True:
        response = wcapi.get('products', params=product_data)
        products = response.json()
        products_formatted = []
        for product in products:
            try:
                product_img_src = product["images"][0]["src"]
            except IndexError:
                product_img_src = "invalid picture src"
            product_stock_status = `product["stock_status"]``

Hello, there is a bug in line110. The String "src" must be an integer in my python compiler.

**Comment by 5rzmz8odb** [comment]
hi @control-led did you solved it?
same bug here @JohannBraun is there an easier way to loop-in and solve the error? 🤔

**Comment by JohannBraun** [comment]
If im not mistaken it's not actual a bug. It only shows an error if you don't get any data from your api call. Are you sure you got a response? Try to print the response that you get from line 106. It shouldnt be empty.

**Comment by 5rzmz8odb** [comment]
You are totally right @JohannBraun there must be an authentification error that I cannot handle, so the api returns no data, will try to fix it, probably is a hosting OAuth incompability problem

Thanks!
